shepherd_bms/src/compute.cpp

```diff
@@ -33,9 +33,9 @@ FaultStatus_t ComputeInterface::sendChargingMessage(uint16_t voltageToSet, uint1
         currentToSet = 10;
     }
     chargerMsg.cfg.chargerCurrent = currentToSet * 10 + 3200;
-    bool is_charging = isCharging();
     
-    chargerMsg.cfg.chargerLEDs = calcChargerLEDState(state_of_charge, is_charging);
+    
+    chargerMsg.cfg.chargerLEDs = calcChargerLEDState(state_of_charge, );
```

Comment:
? wtw on the comma after the first param and then nothing. might be why no build